package.json

```diff
@@ -39,8 +39,10 @@
     "@nestjs/typeorm": "^7.1.4",
     "apollo-server-express": "^2.18.2",
     "document-register-element": "1.13.1",
+    "fs": "0.0.1-security",
```

Comment:
what is this package?

Reply:
Oh I don't think we were aware of an existing package. Which one is it?